Automattic/wp-calypso
# Components: Add QueryBillingTransactions query component

This PR introduces a simple `QueryBillingTransactions` query component that can be used for billing transactions network requests. 

This PR is part of the billing history reduxification effort (#10554).

To test:
* Checkout this branch.
* Insert the query component anywhere.
* Verify it triggers the expected network requests (to `/me/billing-history`).
